### ISSUE TITLE
$comment field for property can be turned into JavaDoc

### ISSUE THREAD

**Issue by markusjevringsesame** [joelittlejohn/jsonschema2pojo#1197]
We use the `$comment` field a lot in our schemas, and it would be nice if the content of that was added as JavaDoc to the fields and setters, getters, withers, etc. Right now the only JavaDoc we get is `(Required)`.

**Comment by eirnym** [comment]
Hi there, 

It's a nice enchantment. Do you mind to make a PR for this?

**Comment by markusjevringsesame** [comment]
I might be able to find some time.

**Comment by eirnym** [comment]
Great! I'll be happy to add this feature into the core tool

**Comment by joelittlejohn** [comment]
Yeah, sounds good to me. 

Check out [TitleRule](https://github.com/joelittlejohn/jsonschema2pojo/blob/master/jsonschema2pojo-core/src/main/java/org/jsonschema2pojo/rules/TitleRule.java). Should be pretty easy to create a CommentRule 👍

**Comment by markusjevringsesame** [comment]
@joelittlejohn @eirnym I'm getting this from the spec for draft-7 about the '$comment' field: "Implementations MUST NOT present this string to end users."
https://tools.ietf.org/html/draft-handrews-json-schema-01#section-9

I also realized that the 'description' field already works exactly the way I thought the '$comment' field worked. What's the recommendation here? Should I still raise a PR for supporting the '$comment' field (now that I finally started working on it it's pretty simple), or should the recommendation just be "use the description field"?
